tests/icons.test.js

```diff
@@ -27,6 +32,17 @@ icons.forEach(icon => {
 
   test(`${icon.title} has an "svg" value`, () => {
     expect(typeof subject.svg).toBe('string');
+    const svgFileContents = fs.readFileSync(svgPath, 'utf8')
+      .replace('" />', '"/>')
+      .replace(/\r?\n/, '')
+      .replace('></path>', '/>')
+      .replace(
+        'xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\">',
+        'viewBox=\"0 0 24 24\" xmlns=\"http://www.w3.org/2000/svg\">')
+      .replace(
+        /<title>([^<]+)<\/title>/,
+        `<title>${titleToHtmlFriendly(subject.title)}</title>`);
```

Comment:
To prevent these replacements the icons contents must be checked before, maybe creating some new SVG lint rules.

Reply:
> maybe creating some new SVG lint rules

Opened a PR for this on #6513

Reply:
I'm ok with merging this and removing them in the follow up PR.